tests/app/main/test_briefs.py

```diff
@@ -120,6 +129,7 @@ def test_clarification_question_checks_supplier_is_eligible(self, data_api_clien
         self.login()
         data_api_client.get_brief.return_value = api_stubs.brief(status='live')
         data_api_client.get_brief.return_value['briefs']['frameworkName'] = 'Digital Outcomes and Specialists'
+        data_api_client.get_brief.return_value['briefs']['lot'] = 'digital-specialists'
```

Comment:
Actually yes the `frameworkName` is also set by default - see https://github.com/alphagov/digitalmarketplace-apiclient/blob/master/dmapiclient/api_stubs.py#L33.  My guess is that in the past when this test was written it didn't.